TheSufferfest/electron-crash-report-server
# WB-275 WB-276 create enpoinds for `/electron-crashes` and `/js-errors`

#### What's this PR do?
Adds 2 new endpoints:
`/electron-crashes` and `/js-errors`
#### Any background context you want to provide?
I'm pointing this to pr/7 branch because that needs to be merge first.
#### How should this be manually tested?
Use this PR https://github.com/TheSufferfest/SufferfestWindows/pull/586 to test the crash posting, point to the both routes to check that a `type` property is added to the body
#### Could any automated tests be added for this?
Maybe
#### What are the relevant JIRA issues?
https://thesufferfest.atlassian.net/browse/WB-275
https://thesufferfest.atlassian.net/browse/WB-276
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/5782744/38581216-c9e84f74-3cd1-11e8-81ec-e7d60fff92cf.png)

![image](https://user-images.githubusercontent.com/5782744/38581229-d668d8ea-3cd1-11e8-8ae5-b106862fbf06.png)

![image](https://user-images.githubusercontent.com/5782744/38581290-097e68da-3cd2-11e8-9088-96eb84997192.png)
